hazelcast/src/test/java/com/hazelcast/config/replacer/EncryptionReplacerTest.java

```diff
@@ -49,7 +47,6 @@
  * Unit tests for {@link EncryptionReplacer}.
  */
 @RunWith(HazelcastSerialClassRunner.class)
```

Comment:
Does this class override the runner to be serial?

Reply:
For some reason it does, not sure why.